WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -151,6 +174,27 @@ class PageListViewModel @Inject constructor() : ViewModel() {
         }
     }
 
+    private fun getFeaturedImageUrl(featuredImageId: Long): String? {
+        if (featuredImageId == 0L) {
+            return ReaderImageScanner("", !SiteUtils.isPhotonCapable(pagesViewModel.site)).largestImage
```

Comment:
Shouldn't we pass the content of the page to the `ReaderImageScanner`? Otherwise this won't really do anything.

Reply:
Oops, I forgot about this. It's just a leftover when I was trying to get it working. But I'm actually wondering whether we should use it. Unlike in posts where you want to show some image from the post content to better identify it in the list (even if it's not explicitly set as a featured image), I'm not sure it makes sense for pages. What do you think, @SylvesterWilmott?

Reply:
For pages I think it makes more sense to only show featured image. This is what was implemented on iOS as well